roles/gpu_operator_wait_deployment/files/fetch_pod_metrics.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -e
+
+if [[ "${1:-}" == "" || "${1}" == "-h" || "${1}" == "--help" ]]; then
+    cat <<EOF
+Fetch metrics from a Pod
+
+Usage: $0 <POD_PORT> <LABEL_SELECTOR> <NAMESPACE> <PROM_GREP_KEY>
+Example:
+  $0 9400 app=nvidia-dcgm-exporter gpu-operator-resources '#HELP'
+EOF
+    exit 0
+fi
+
+POD_PORT="$1"
+LABEL_SELECTOR="$2"
+NAMESPACE="$3"
+PROM_GREP_KEY="$4"
+
+LOCAL_PORT=9401
+
+pod_name=$(oc get pods -oname -l$LABEL_SELECTOR -n$NAMESPACE | head -1);
+if [ -z "$pod_name" ]; then
+    echo "Failed to find a pod for $LABEL_SELECTOR in namespace $NAMESPACE";
+    exit 10;
+fi;
+
+retry=5;
+timeout 10 oc port-forward ${pod_name} ${LOCAL_PORT}:${POD_PORT} -n $NAMESPACE &
+trap wait EXIT
+pod_output=""
+while [ "$pod_output" == "" ]; do
+    sleep 1;
+    pod_output=$(curl localhost:${LOCAL_PORT}/metrics 2>/dev/null);
```

Comment:
Since this is a proper script now, I think all those `;` are unnecessary

Reply:
ack, it was a left-over from when I moved it from ansible task :)